.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "subprojects/xutils-dev"]
 	path = subprojects/xutils-dev
 	url = https://salsa.debian.org/xorg-team/app/xutils-dev.git
+[submodule "subprojects/winetricks"]
+	path = subprojects/winetricks
+	url = https://github.com/Winetricks/winetricks.git
```

Comment:
Winetricks is moving pretty slow and the releases are getting quite stale. We want to use the latest, I think.

The submodules are actually fixed on a specific commit though, so we don't need to use the latest. There's really no difference from the current solution.

---

I'm working on a reimplementation of Winetricks in Python. I can't give an ETA at the moment, but I have some groundwork and I'm working on an importer that converts the verbs into a maintainable file format.

Winetricks is huge though... and slow. It has a lot of functionality, so it will take some time to implement all of it.

I hope to finish work on it, but don't hold your breath. I'm not sure if I'm motivated enough to finish this little side project.

Reply:
We can actually override verbs for small fixes by adding them to the local verbs. So we don't need to fork.

Reply:
> I'm working on a reimplementation of Winetricks in Python. I can't give an ETA at the moment, but I have some groundwork and I'm working on an importer that converts the verbs into a maintainable file format.
> 
> Winetricks is huge though... and slow. It has a lot of functionality, so it will take some time to implement all of it.
> 
> I hope to finish work on it, but don't hold your breath. I'm not sure if I'm motivated enough to finish this little side project.

This is great. Yes, I have been deliberating about a winetricks rewrite for a bit and had kinda planned for this, actually. Like the rewrite would be tailored to Proton, only containing the most useful verbs and metadata of them would be separated from the main program to separate files (e.g., toml, json, etc.). And while I imagine a rewrite in Python would speed up its runtime execution, be simpler, and better designed, I believe only part of its execution would be sped up unless we do not create subprocesses to wine binaries like the original script or perhaps if the registration of DLLs is possible to do in a thread-safe manner (which I haven't really explored yet).

Reply:
I think it would be modules that can be imported with a CLI that imports the modules as well. Since the verbs would be in individual files, it doesn't really matter if there are only the most useful verbs or all of them. I will implement the most used functions of the verbs first, which should port most of the verbs pretty fast.

> I believe only part of its execution would be sped up unless we do not create subprocesses to wine binaries like the original script or perhaps if the registration of DLLs is possible to do in a thread-safe manner (which I haven't really explored yet).

This is an issue, but I think some verbs are overkill.. like installing a complete software suite for having a dll.. but to refactor and test those verbs, we would need way more manpower. Not realistic atm.. but registering dlls or changing the registry might be viable.